### PR TITLE
Feature/geode 3795

### PR DESCRIPTION
**Pull request by galen-pivotal**
Move everything to a new package so that we can version the protocol better later.

Also ran IntelliJ's "optimize imports" on the `geode-protobuf` package. It may be easier to read the two commits separately.

---
- [x] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [x] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [even better] Is your initial contribution a single, squashed commit?

- [x] Does `gradlew build` run cleanly?

- [x] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by galen-pivotal** [comment]
@PivotalSarge @WireBaron

File: geode-protobuf/src/main/proto/org/apache/geode/internal/protocol/protobuf/v1/clientProtocol.proto [hunk 1]
```diff
@@ -19,13 +19,13 @@
...
 
 import "google/protobuf/any.proto";
-import "region_API.proto";
-import "server_API.proto";
-import "basicTypes.proto";
-import "connection_API.proto";
+import "org/apache/geode/internal/protocol/protobuf/v1/region_API.proto";
+import "org/apache/geode/internal/protocol/protobuf/v1/server_API.proto";
+import "org/apache/geode/internal/protocol/protobuf/v1/basicTypes.proto";
+import "org/apache/geode/internal/protocol/protobuf/v1/connection_API.proto";
```
Comment (WireBaron) [inline geode-protobuf/src/main/proto/org/apache/geode/internal/protocol/protobuf/v1/clientProtocol.proto]: I see what you were saying about how these includes got a lot more verbose.

File: geode-protobuf/src/test/java/org/apache/geode/internal/protocol/OpsHandler.java [hunk 2]
```diff
@@ -1,21 +0,0 @@
...
- * or implied. See the License for the specific language governing permissions and limitations under
- * the License.
- */
-package org.apache.geode.internal.protocol;
-
-import org.apache.geode.internal.protocol.serialization.registry.SerializationCodecRegistry;
-
-public interface OpsHandler<Req, Resp> {
-  Resp process(SerializationCodecRegistry serializationCodecRegistry, Req request);
-}
```
Comment (WireBaron) [inline geode-protobuf/src/test/java/org/apache/geode/internal/protocol/OpsHandler.java]: Hmm, and this class also gets a big WTF?  Must have been some cruft that got left behind when someone blew away whatever tests were using it (probably even odds that was my bad).

File: geode-protobuf/src/main/proto/org/apache/geode/internal/protocol/protobuf/v1/clientProtocol.proto [hunk 3]
```diff
@@ -19,13 +19,13 @@
...
 syntax = "proto3";
-package org.apache.geode.internal.protocol.protobuf;
+package org.apache.geode.internal.protocol.protobuf.v1;
 
 import "google/protobuf/any.proto";
-import "region_API.proto";
-import "server_API.proto";
-import "basicTypes.proto";
-import "connection_API.proto";
+import "org/apache/geode/internal/protocol/protobuf/v1/region_API.proto";
```
Comment (kohlmu-pivotal) [inline geode-protobuf/src/main/proto/org/apache/geode/internal/protocol/protobuf/v1/clientProtocol.proto]: I don't understand why the proto file location has to be fully qualified. I've tested and just having the files reside in `v1/*proto` works

**Comment by galen-pivotal** [comment]
@kohlmu-pivotal I've tried running the protobuf compiler with the files in `src/main/proto/v1`, but the compiler fails to find the other files. Same thing goes for the now-longer import statements.

**Pull request merged**